### ISSUE TITLE
unknown extra/redisotel and extra/rediscmd versions

### ISSUE THREAD

**Issue by vadimi** [redis/go-redis#3078]
Hello, it looks like `extra/rediscmd/v9.5.3` and `extra/redisotel/v9.5.3` tags were removed from the repo, but those versions are still listed in https://pkg.go.dev/github.com/redis/go-redis/extra/rediscmd/v9@v9.5.3?tab=versions and https://pkg.go.dev/github.com/redis/go-redis/extra/redisotel/v9?tab=versions. This breaks some automated package version updates in CI environments.

I think it is related to https://github.com/redis/go-redis/issues/3057

**Comment by sysulq** [comment]
Seems like `extra/rediscmd` and `extra/redisotel` only have upper version `v9.0.5`, lacks version for v9.5.x or v9.6.x... Maybe we should add new tags for these versions?

**Comment by vadimi** [comment]
new tags would be great and also retract version `v9.5.3`

**Comment by ipechorin** [comment]
Hi, any update on this issue please? Is there anything preventing from adding new tags for `extra/rediscmd` and `extra/redisotel`? After v9.5.3 was retracted, the latest version available for these two is just `v9.0.5` which is 1.5 years old now...

**Comment by hoffm** [comment]
After much confusion, we are also forced to downgrade to v9.0.5 because these tags were deleted. It would be great to get newer tags, or at least to know what changes we losing in the downgrade. 

Note that pkg.go.dev still [lists](https://pkg.go.dev/github.com/redis/go-redis/extra/redisotel/v9?tab=versions) the retracted versions of these packages, which adds to the confusion.

![redisotel package versions - github comredisgo-redisextraredisotelv9 - Go Packages 2024-09-06 at 9 58 28 AM](https://github.com/user-attachments/assets/9ffe48e6-c24d-4b93-85b7-43bf8119b19b)

**Comment by vadimi** [comment]
`v9.5.3` of `redisotel` (and other `extra` packages) wasn't really retracted - there needs to be a change added to https://github.com/redis/go-redis/blob/master/extra/redisotel/go.mod to get it done and new version/tags have to be released.

**Comment by ipechorin** [comment]
https://github.com/redis/go-redis/pull/3108

**Comment by vadimi** [comment]
hello @ofekshenawa @vladvildanov @dmaier-redislabs , apologies for tagging you directly, but any chance the versions of `extra` packages can be fixed?

**Comment by vadimi-ep** [comment]
@vladvildanov I see `v9.6.2` got released. Any chance you could tag `v9.6.2` or `extra` packages? 

<img width="308" alt="image" src="https://github.com/user-attachments/assets/4754beeb-8d15-4394-90a5-3c957e74917a">

**Comment by vladvildanov** [comment]
@vadimi-ep Thanks for pointing us out, new tags has been added and now will be a part of each release

**Comment by ipechorin** [comment]
@vladvildanov Thanks for adding the tags! I am afraid there's still an issue with redisotel: redisotel v9.6.2 references non-existing version v9.6.1 of rediscmd?

> $ go get github.com/redis/go-redis/extra/redisotel/v9@v9.6.2
go: github.com/redis/go-redis/extra/redisotel/v9@v9.6.2 requires github.com/redis/go-redis/extra/rediscmd/v9@v9.6.1: reading github.com/redis/go-redis/extra/rediscmd/go.mod at revision extra/rediscmd/v9.6.1: unknown revision extra/rediscmd/v9.6.1

**Comment by vadimi** [comment]
@vladvildanov thanks for pushing new tags, I'm having the same issue updating to 9.6.2 as @ipechorin above

**Comment by vladvildanov** [comment]
@vadimi Thanks for your patience guys! I see that Go pkg didn't pick a tag for redisotel. We're on it currently.

**Comment by vadimi** [comment]
it looks like `v9.6.2` is broken beyond repair due to go pkg proxy caching. `v9.6.3` of `extra` packages might be needed at this point.

**Comment by vladvildanov** [comment]
@vadimi Could try now?

**Comment by vladvildanov** [comment]
@vadimi For some reason, package manager doesn't show the version on the website. But if you pull with `go get github.com/redis/go-redis/extra/rediscmd/v9`, latest will point to `v9.6.2`

**Comment by vadimi** [comment]
@vladvildanov same error:
```
go get github.com/redis/go-redis/extra/rediscmd/v9
go: github.com/redis/go-redis/extra/rediscmd/v9@v9.6.1: reading github.com/redis/go-redis/extra/rediscmd/go.mod at revision extra/rediscmd/v9.6.1: unknown revision extra/rediscmd/v9.6.1
```

go pkg proxy will not let you "republish" the same version with a different commit sha, unfortunately you'd need to release `v9.6.3` or extra packages

**Comment by vladvildanov** [comment]
@vadimi We just released 9.7.0 and now it's a latest for all extra packages, I hope it will work for you

**Comment by vadimi** [comment]
thanks @vladvildanov, it's working for me now 🎉
